app/controllers/books_controller.rb

```diff
@@ -13,6 +13,6 @@ def show
 
   def rents
     @rents = Book.find(params[:id]).rents
-    render_paginated @rents, each_serializer: RentSerializer
+    render_paginated @rents
```

Comment:
In this particular case you want to specify the serializer (if it's a collection)